tests/test_updater_consistent_snapshot.py

```diff
@@ -197,36 +199,32 @@ def test_download_targets(self, test_case_data: Dict[str, Any]):
         consistent_snapshot: bool = test_case_data["consistent_snapshot"]
         prefix_targets_with_hash: bool = test_case_data["prefix_targets"]
         hash_algo: Optional[str] = test_case_data["hash_algo"]
-        targetpaths = ["file", "file.txt", "..file.ext", "f.le"]
+        targetpaths: List[str] = test_case_data["targetpaths"]
```

Comment:
is there an advantage to this change?

Reply:
If combined with your comment below, I would say readability (?) adding the `targetpaths` to the test dataset.